lib/rules/accessor-pairs.js

```diff
@@ -66,80 +66,84 @@ function isPropertyDescriptor(node) {
 // Rule Definition
 //------------------------------------------------------------------------------
 
-module.exports = function(context) {
-    var config = context.options[0] || {};
-    var checkGetWithoutSet = config.getWithoutSet === true;
-    var checkSetWithoutGet = config.setWithoutGet !== false;
-
-    /**
-     * Checks a object expression to see if it has setter and getter both present or none.
-     * @param {ASTNode} node The node to check.
-     * @returns {void}
-     * @private
-     */
-    function checkLonelySetGet(node) {
-        var isSetPresent = false;
-        var isGetPresent = false;
-        var isDescriptor = isPropertyDescriptor(node);
-
-        for (var i = 0, end = node.properties.length; i < end; i++) {
-            var property = node.properties[i];
-
-            var propToCheck = "";
-            if (property.kind === "init") {
-                if (isDescriptor && !property.computed) {
-                    propToCheck = property.key.name;
-                }
-            } else {
-                propToCheck = property.kind;
+module.exports = {
+    meta: {
+        errors: {
+            getterMissing: "Getter is not present",
+            setterMissing: "Setter is not present"
+        },
+        description: "enforces getter/setter pairs in objects",
+        ruleType: "Best Practices",
+        recommended: false,
+        fixable: false,
```

Comment:
Hmm.. What if we update `source-code-fixer` to check for this before applying fixes? Not sure I like it, since it's just another obstacle for rule developers, for no apparent reason, other then to keep metadata in sync.